prow/plugins/label/label.go

```diff
@@ -117,6 +118,21 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 	return
 }
 
+func (ae assignEvent) getRepeats(sigMatches [][]string, existingLabels map[string]string) (toRepeat []string) {
```

Comment:
This repeats a lot of the functionality from the for loop below that actually adds the labels.  That said, it order to make it modular and testable, this is what I came up with